engine/src/flutter/shell/common/shorebird/shorebird.cc

```diff
@@ -86,10 +86,30 @@ bool ConfigureShorebird(const ShorebirdConfigArgs& args,
   patch_path = args.release_app_library_path;
   auto shorebird_updater_dir_name = "shorebird_updater";
 
+  // Parse app id from shorebird.yaml
```

Comment:
I would split this out and write a brief unittest for it.  shorebird_test.cc should exist and can be used.

string ReadYamlKey(string yaml);